joomla/joomla-cms
# Messages inside the new Joomla 3.6 updater / max upload size is lower than 8mb

Pull Request for Issue #10604 
#### Summary of Changes

change extensions to packages
#### Testing Instructions
- set maximum upload file size to less than 8MB
- see the message in the updater
- see the message in the extension manger
